llmfoundry/callbacks/hf_checkpointer.py

```diff
@@ -279,6 +279,14 @@ def _save_checkpoint(self, state: State, logger: Logger):
                             path=local_save_path,
                             **self.mlflow_logging_config,
                         )
+
+                        if os.path.exists(
+                                os.path.join(local_save_path, 'LICENSE.txt')):
+                            mlflow_logger._mlflow_client.log_artifact(
+                                mlflow_logger._run_id,
+                                os.path.join(local_save_path, 'LICENSE.txt'),
+                            )
```

Comment:
You know what would be sick here? a walrus operator

Reply:
```
if (license_path := os.path.join(local_save_path, 'LICENSE.txt')) and os.path.exists(license_path):
    mlflow_logger._mlflow_client.log_artifact(
        mlflow_logger._run_id,
        license_path,
    )

```

Reply:
or I guess u could just move the path creation to a new line. That's probably better tbh

Reply:
oh yeah, this should just be a variable.